templates/api.hbs

```diff
@@ -32,7 +54,9 @@ export default function(app: Express, impl: t.{{className name}}Api) {
 						{{#unless schema}}
 						return undefined
 						{{else}}
-						{{#ifmatch mediaType.mimeType '/json'}}
+						{{#ifmatch mediaType.mimeType 'multipart/form-data'}}
+						return {{>frag/fromMultipart schema prefix='v.'}}('body', req)
+						{{else if (ifmatch mediaType.mimeType '/json')}}
```

Comment:
I'd like to use `(isContentJson .)` here instead of `(ifmatch mediaType.mimeType '/json')`